modules/scripts/startup-script/variables.tf

```diff
@@ -112,15 +112,16 @@ variable "install_ansible" {
   default     = null
 }
 
-variable "configure_ssh" {
+variable "configure_ssh_host_prefix" {
```

Comment:
Prefix seems like incorrect name now that you are not adding anything to the end. Maybe `configure_ssh_host_patterns`.

Reply:
Yes, this is much better indeed.